settings.py

```diff
@@ -347,4 +345,15 @@
         'social_core.pipeline.user.user_details',
         'seqr.utils.social_auth_pipeline.log_signed_in',
     )
-    INSTALLED_APPS.append('social_django')
+else:
+    SOCIAL_AUTH_PIPELINE = (
+        'social_core.pipeline.social_auth.social_details',
+        'social_core.pipeline.social_auth.social_uid',
+        'social_core.pipeline.social_auth.social_user',
+        'social_core.pipeline.user.get_username',
+        'social_core.pipeline.social_auth.associate_by_email',
+        'social_core.pipeline.social_auth.associate_user',
+        'social_core.pipeline.social_auth.load_extra_data',
+        'social_core.pipeline.user.user_details',
+        'seqr.utils.social_auth_pipeline.log_signed_in',
+    )
```

Comment:
theres a lot of copy-paste here, which makes it hard to see the clear differences between the pipeline in the two cases. it might be helpful if you defined it and then edit it to make it work in the terra case, so something like

```
SOCIAL_AUTH_PIPELINE = [
   ...
]
if TERRA_API_ROOT_URL:
   ...
   SOCIAL_AUTH_PIPELINE.insert(0, 'seqr.utils.social_auth_pipeline.validate_anvil_registration')
   SOCIAL_AUTH_PIPELINE.insert(6, 'social_core.pipeline.user.create_user')
```

Reply:
Sounds good.